src/components/MultipleAvatars.js

```diff
@@ -10,58 +10,86 @@ const propTypes = {
 
     // Whether this option is currently in focus so we can modify its style
     optionIsFocused: PropTypes.bool,
+
+    // Set the sie of avatars
+    size: PropTypes.oneOf(['default', 'small']),
+
+    // Styles to override the basic component styles
+    styles: PropTypes.shape({
+        // Style for First Avatar on Multiple Avatars
+        // eslint-disable-next-line react/forbid-prop-types
+        singleAvatar: PropTypes.object,
+
+        // Style for Second Avatar on Multiple Avatars
+        // eslint-disable-next-line react/forbid-prop-types
+        secondAvatar: PropTypes.object,
+
+        // Style for avatar Container
+        // eslint-disable-next-line react/forbid-prop-types
+        emptyAvatar: PropTypes.object,
```

Comment:
I'm not following where we are getting these other style props from...

`styles.singleAvatar` and `styles.emptyAvatar`

I don't see where we are passing them as props.

Reply:
Also we should not be passing style objects as props anyway unless it is an array or single object of styles.